README.md

```diff
@@ -22,8 +58,10 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Microverse
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
 
 ## 📝 License
 
-This project is [MIT](./LICENSE) licensed.
\ No newline at end of file
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] As requested by the previous reviewer, all links should work.  This needs to point to your LICENSE file.  Kindly make it so.

That way, your project will be well documented 💪